docs/app-check/debug-provider.md

```diff
@@ -20,63 +20,30 @@ app, and don't share your debug builds with untrusted parties.
 
 ## Apple platforms
 
-To use the debug provider while running your app in a simulator interactively
+To use the debug provider (default is device check provider) while running your app in a simulator interactively
 (during development, for example), do the following:
 
-1.  In the file `ios/Runner/AppDelegate.swift` (or `AppDelegate.m`), create and
```

Comment:
If the user has this already and switches to this new version and setup then will it conflict/be a breaking change?

Reply:
As far as my testing goes, this is initialised after we have already initialised (double checked with break points) in the plugin and will be ignored by App Check iOS.

I also tested by calling `activate()` with deviceCheck provider which didn't show up any failed API requests.

I then called `activate()` with the debug provider (updates the facade instance on the fly), and received failed API requests (this only appears to happen on debug provider).

I switched back to deviceCheck provider on the fly, and the API request were no longer failing.

This is evidence that the initialisation occurring in the AppDelegate is ignored. Although, ideally, the user would remove any setup code in the AppDelegate.

Reply:
Would be nice to let user know to remove this code, but wondering about best way to do that 🤔